### PR TITLE
Fix install image and other issues

### PR DESCRIPTION
**Pull request by neelimamukiri**
Fixes the following issues:
https://github.com/contiv/install/issues/162
https://github.com/contiv/install/issues/161
https://github.com/contiv/install/issues/154
https://github.com/contiv/install/issues/149
https://github.com/contiv/install/issues/141

Also add the full installer back (hit another lab setup where
curl to get the netplugin binaries is timing out). Updated
the corresponding release script and the "," fix for env.json issue.

### CODE REVIEW

File: install/ansible/install.sh [hunk 1]
```diff
@@ -140,19 +140,24 @@ if [ "$aci_image" != "" ]; then
...
 	sed -i.bak "s#.*contiv_v2plugin_install.*#\"contiv_v2plugin_install\":\"True\",#g" "$env_file"
+else
+	sed -i.bak "s#.*contiv_v2plugin_install.*#\"contiv_v2plugin_install\":\"False\",#g" "$env_file"
 fi
 
 echo "Installing Contiv"
 # Always install the base, install the scheduler stack/etcd if required
-echo '- include: install_base.yml' >$ansible_path/install_plays.yml
+
+rm -rf $ansible_path/install_plays.yml
```
Comment (dseevr) [inline install/ansible/install.sh]: Technically this doesn't need -r

File: install/ansible/uninstall.sh [hunk 2]
```diff
@@ -135,16 +135,17 @@ sed -i.bak "s/.*docker_reset_container_state.*/\"docker_reset_container_state\":
...
 fi
-if [ "$contiv_v2plugin_install" == "true" ]; then
+if [ "$uninstall_v2plugin" == "true" ]; then
 	sed -i.bak "s#.*contiv_v2plugin_install.*#\"contiv_v2plugin_install\":\"True\",#g" "$env_file"
+else
+	sed -i.bak "s#.*contiv_v2plugin_install.*#\"contiv_v2plugin_install\":\"False\",#g" "$env_file"
 fi
 
 echo "Uninstalling Contiv"
+rm -rf $ansible_path/uninstall_plays.yml
```
Comment (dseevr) [inline install/ansible/uninstall.sh]: Here too

File: scripts/build.sh [hunk 3]
```diff
@@ -126,7 +126,32 @@ mkdir -p $binary_cache
...
+curl -sL -o $binary_cache/openvswitch-2.5.0-2.el7.x86_64.rpm http://cbs.centos.org/kojifiles/packages/openvswitch/2.5.0/2.el7/x86_64/openvswitch-2.5.0-2.el7.x86_64.rpm
+curl -sL -o $binary_cache/ovs-common.deb http://mirrors.kernel.org/ubuntu/pool/main/o/openvswitch/openvswitch-common_2.5.2-0ubuntu0.16.04.1_amd64.deb
+curl -sL -o $binary_cache/ovs-switch.deb http://mirrors.kernel.org/ubuntu/pool/main/o/openvswitch/openvswitch-switch_2.5.2-0ubuntu0.16.04.1_amd64.deb
+curl -sL -o $binary_cache/netplugin-$contiv_version.tar.bz2 https://github.com/contiv/netplugin/releases/download/$contiv_version/netplugin-$contiv_version.tar.bz2
+
+env_file=$output_dir/install/ansible/env.json
+sed -i.bak "s#.*auth_proxy_local_install.*#  \"auth_proxy_local_install\": True,#g" $env_file
+sed -i.bak "s#.*contiv_network_local_install.*#  \"contiv_network_local_install\": True,#g" $env_file
+
+# Create the full tar bundle
```
Comment (gaurav-dalvi) [inline scripts/build.sh]: So we are going to create two builds full and minimal again ?

Comment (dseevr) [inline scripts/build.sh]: Yep

**Pull request merged**